src/app/core/pages/page-not-found/page-not-found.component.html

```diff
@@ -1,5 +1,5 @@
 <p i18n>This page does not exist!</p>
 <p>
   <span i18n>Go back to the </span>
-  <a routerLink="/" i18n>home page</a>
```

Comment:
I noticed that the not found page hides the left nav, it's due to another task of mine. Can you add the following to `page-not-found.component` :pray: :
```ts
  constructor(
    private layoutService: LayoutService,
  ) {
    this.layoutService.configure({ fullFrameActive: false });
  }
```

![image](https://user-images.githubusercontent.com/16257159/163212975-e283f764-eb60-4810-a23e-c40208c18560.png)

Reply:
Fixed